etc/helm/pachyderm/templates/_helpers.tpl

```diff
@@ -163,6 +163,16 @@ http://localhost:30658
 {{- end }}
 {{- end }}
 
+{{- define "oidc.hasRealIDP" -}}
+{{- if .Values.oidc.upstreamIDPs -}}
+true
+{{- else if .Values.oidc.mockIDP -}}
```

Comment:
simplify to `else false`?

Reply:
it's redundant but I wanted to call out `mockIDP` - I'll remove it if you prefer